### ISSUE TITLE
Remember manual forecast-matching when confirming your hours

### ISSUE THREAD

**Issue by tarjeieo** [Puzzlepart/did#572]
- [X] I have searched for existing issues to avoid creating duplicates

### TLDR
Manually matching appointments in the forecast feature does not match them when confirming the week week(s) later. Can we somehow remember the previous matching?

### Description

With the new forecast feature, you sometimes have to manually resolve some appointments. When week(s) later you are confirming your week, many of the appointments will be the same, and therefore you need to match them again. Obviously this is another argument for using categories and matching directly in the appointment, but anyway - is there a way to remember the preferred matching when confirming the week?
